addons/hr_timesheet/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,39 @@
+---Fields in module 'hr_timesheet'---
+hr_timesheet / account.analytic.line    / is_timesheet (boolean)        : NEW 
+#TODO : Set all analytic line to true where its analytic account use_timesheet is true.
```

Comment:
delete those when it's done (rationale: We don't want false positives when we grep for TODO in the work files)